src/wizard-settings/header-bidding-gam/index.js

```diff
@@ -224,56 +124,125 @@ const HeaderBiddingGAM = () => {
+			{ isManaging && editingOrder === false && (
 				<Modal
-					title={ __( 'Create Order', 'newspack-ads' ) }
-					onRequestClose={ () => ! inFlight && setIsCreating( false ) }
+					title={ __( 'Manage Orders', 'newspack-ads' ) }
+					onRequestClose={ () => ! inFlight && setIsManaging( false ) }
 				>
-					<p>
-						{ __(
-							'Create the order and line items on your Google Ad Manager network according to the pre-defined price bucket settings.',
-							'newspack-ads'
-						) }
-					</p>
-					{ error && error.data?.status !== '404' && (
-						<Notice isError noticeText={ error.message } />
+					{ activeOrders.length && (
+						<>
+							<Card noBorder>
+								{ activeOrders.map( order => (
+									<ActionCard
+										key={ order.id }
+										title={ order.name }
+										badge={ order.status }
+										description={ () => (
+											<span>
+												{ sprintf(
+													// Translators: the bidder revenue share for this order.
+													__( 'Bidder Revenue Share: %1$d%%', 'newspack-ads' ),
+													order.revenue_share || 0
+												) }{ ' ' }
+												|{ ' ' }
+												{ sprintf(
+													// Translators: comma-separated list of adapters for the order or "any" if undefined.
+													__( 'Bidders: %s', 'newspack-ads' ),
+													order.bidders?.length
+														? order.bidders
+																.map( bidderKey => bidders[ bidderKey ]?.name || bidderKey )
+																.join( ', ' )
+														: __( 'any', 'newspack-ads' )
+												) }
+											</span>
+										) }
+										actionText={
+											<OrderPopover
+												isDraft={ order.status === 'DRAFT' }
+												disabled={ inFlight }
+												onArchive={ async () => {
+													if (
+														// eslint-disable-next-line no-alert
+														confirm(
+															__( "Are you sure you'd like to archive this order?", 'newspack-ads' )
+														)
+													) {
+														setInFlight( true );
+														await archiveOrder( order.id );
+														setInFlight( false );
+														await fetchOrders();
+													}
+												} }
+												onEdit={ () => setEditingOrder( order.id ) }
+												gamLink={ getOrderUrl( order.id ) }
+											/>
+										}
+										className="mv0"
+										isSmall
+									/>
+								) ) }
+							</Card>
+
+							{
+								// Display warning if a bidder is being targeted by more than one order.
+								Object.keys( bidders ).map( bidderKey => {
+									const bidderOrders = activeOrders.filter( order =>
+										order.bidders?.includes( bidderKey )
+									);
+									return (
+										bidderOrders.length > 1 && (
+											<Notice
+												key={ bidderKey }
+												isWarning
+												noticeText={ sprintf(
+													// Translators: %1 The name of the bidder. %2 The number of orders for the bidder.
+													__( '%1$s is being targeted by %2$d orders.', 'newspack-ads' ),
```

Comment:
Question, not issue: I'm not clear on the difference between an order that targets all bidders (all bidders selected) vs. one that targets "any" (no bidders selected). In the latter case should we also show this warning if there's another order targeting a bidder?

Reply:
Empty or all selected produce the same effect. Although we support creating orders with any bidder configuration, the warnings should help control potential conflicts with the setup. 1a84e881c9d99141a267da3cfd0ecdd1dfad4dac updates the warning to consider "empty" bidder configuration as "any".

Reply:
Confirmed, thanks for clarifying!